src/error/mod.rs

```diff
@@ -52,6 +58,8 @@ impl Error {
             Error::SignatureValidationError(_) => StatusCode::BAD_REQUEST,
             Error::Base64Error(_) => StatusCode::BAD_REQUEST,
             Error::UuidParseError(_) => StatusCode::BAD_REQUEST,
+            Error::UrlParseError(_) => StatusCode::BAD_REQUEST,
+            Error::ArweaveError(_) => StatusCode::BAD_REQUEST,
```

Comment:
Basiclly Arweave upload error is not user's fault, so it should be `5xx`, like `INTERNAL_SERVER_ERROR`